GDJS/Runtime/events-tools/inputtools.js

```diff
@@ -206,6 +206,14 @@ gdjs.evtTools.input.getMouseWheelDelta = function(runtimeScene) {
     return runtimeScene.getGame().getInputManager().getMouseWheelDelta();
 };
 
+gdjs.evtTools.input.ScrollUp = function(runtimeScene) {
+    return runtimeScene.getGame().getInputManager().ScrollUp();
```

Comment:
You have to implement these functions in inputmanager.js too :) 
Also prefer calling the functions isScrollingUp/isScrollingDown.